handlers/meshery_application_handler.go

```diff
@@ -311,8 +315,8 @@ func (h *Handler) handleApplicationPOST(
 			http.Error(rw, ErrEncoding(err, obj).Error(), http.StatusInternalServerError)
 			return
 		}
-
-		err = provider.SaveApplicationSourceContent(token, (mesheryApplicationContent[0].ID).String(), mesheryApplication.SourceContent)
+		savedApplicationID = mesheryApplicationContent[0].ID
```

Comment:
@Revolyssup Sending saved ID (nil if skip-save) along with the pattern. Does this look good?